docs/api/sql/geography/Constructor.md

```diff
@@ -18,3 +18,104 @@
  -->
 
 ## ST_GeogFromWKT
+
+Introduction: Construct a Geography from WKT. If SRID is not set, it defaults to 0 (unknown).
+
+!!!note  The output WKT string uses the FIXED precision model, but all computations are performed using JTS’s default 16-digit precision model. Users can specify a different precision model when converting to a string if needed.
+
+Format:
+
+`ST_GeogFromWKT (Wkt: String)`
+
+`ST_GeogFromWKT (Wkt: String, srid: Integer)`
+
+Since: `v1.8.0`
+
+SQL Example
+
+```sql
+SELECT ST_GeogFromWKT('LINESTRING (1 2, 3 4, 5 6)')
+```
+
+Output:
+
+```
+LINESTRING (1 2, 3 4, 5 6)
+```
+
+SQL Example:
+
+```sql
+SELECT ST_GeogFromWKT('GEOMETRYCOLLECTION (POINT (50 50), LINESTRING (20 30, 40 60, 80 90), POLYGON ((35 15, 45 15, 40 25, 35 15), (30 10, 40 20, 30 20, 30 10)))')
+```
+
+Output:
+
+```
+GEOMETRYCOLLECTION (POINT (50 50), LINESTRING (20 30, 40 60, 80 90), POLYGON ((35 15, 45 15, 40 25, 35 15), (30 10, 40 20, 30 20, 30 10)))
+```
+
+## ST_GeogFromEWKT
+
+Introduction: Construct a Geography from OGC Extended WKT.
+
+Format:
+`ST_GeogFromEWKT (EWkt: String)`
+
+Since: `v1.8.0`
+
+SQL example:
+
+```sql
+SELECT ST_GeogFromEWKT('SRID=4326;LINESTRING (0 0, 3 3, 4 4)')
+```
+
+Output:
+
+```
+LINESTRING (0 0, 3 3, 4 4)
+```
+
+
+## ST_GeogFromWKB
```

Comment:
Please don't put the doc of other function in this PR. Please only keep ST_GeogFromEWKT, and ST_GeogFromWKT